src/pages/home/report/ReportActionsView.js

```diff
@@ -221,12 +224,28 @@ class ReportActionsView extends React.Component {
         needsLayoutCalculation,
     }) {
         return (
-            <ReportActionItem
-                action={item.action}
-                displayAsGroup={this.isConsecutiveActionMadeByPreviousActor(index)}
-                onLayout={onLayout}
-                needsLayoutCalculation={needsLayoutCalculation}
-            />
+            <Hoverable>
+                {isHovered => (
```

Comment:
Maybe this should be `hovered` to more closely match the `Pressable` API if this is what we are working around.